README.md

```diff
@@ -2,6 +2,25 @@
 
 This repo provides an open source app called Offline Mapbook built in WPF with the [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/). With very little customization, you can change the app to use your own data. Screenshots of the app and more detailed documentation can be found on the [Developers Site](https://developers.arcgis.com/example-apps/mapbook-wpf/).
 
+<!-- MDTOC maxdepth:6 firsth1:0 numbering:0 flatten:0 bullets:1 updateOnSave:1 -->
+
+- [Features](#features)   
+- [Detailed Documentation](#detailed-documentation)   
+- [Development Instructions](#development-instructions)   
+   - [Fork the repo](#fork-the-repo)   
+   - [Clone the repo](#clone-the-repo)   
+      - [Command line Git](#command-line-git)   
+      - [Configuring a Remote for a Fork](#configuring-a-remote-for-a-fork)   
+   - [Requirements](#requirements)   
+   - [Resources](#resources)   
+   - [Issues](#issues)   
+   - [Contributing](#contributing)   
+- [MDTOC](#mdtoc)   
+   - [Licensing](#licensing)   
```

Comment:
Actually, it just occurred to me that we probably want all of these to be h2 (`##`). I don't think they're children of 'Development Instructions' and we also don't want 'Licensing' to appear as though it's a child of 'MDTOC'.